translations/messages/nl-be.yaml

```diff
@@ -31,6 +33,7 @@ messages:
     highestversionAvailable: "De laatste versie is v{version}"
     formIsValid: "Je formulier is correct"
     formIsInvalid: "Je formulier is niet correct. Vul al de velden in."
+    previewNotUpdatedBecauseOfErrorsInTtl: "Preview niet geupdate door errors in de ttl code"
```

Comment:
geüpdatet :) als we correct willen zijn, of bijgewerkt

Reply:
geüpdate ;p